### PR TITLE
[tests-only][full-ci]Improve then steps usages inside acceptance tests suite `apiShareMangementToShares`

### PR DESCRIPTION
**Pull request by SagarGi**
## Description
Improve then steps for suite `apiShareManagementToShares`

## Related Issue

- Part of https://github.com/owncloud/core/issues/39512

## How Has This Been Tested?

- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiShareManagementToShares/mergeShare.feature [hunk 1]
```diff
@@ -92,13 +108,19 @@ Feature: sharing
...
     # The server "merges" the 2 shares and presents them to Brian as a single folder inside the "Shares" folder
-    And user "Alice" shares folder "/merge-test-outside-groups-renamebeforesecondshare" with user "Brian" using the sharing API
-    And user "Brian" accepts share "/merge-test-outside-groups-renamebeforesecondshare" offered by user "Alice" using the sharing API
-    Then as "Brian" folder "/Shares/merge-test-outside-groups-renamebeforesecondshare" should exist
+    When user "Alice" shares folder "/merge-test-outside-groups-renamebeforesecondshare" with user "Brian" using the sharing API
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
+    When user "Brian" accepts share "/merge-test-outside-groups-renamebeforesecondshare" offered by user "Alice" using the sharing API
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiShareManagementToShares/mergeShare.feature:122]: why not check the status codes for both requests at the same time?

File: tests/acceptance/features/apiShareManagementToShares/mergeShare.feature [hunk 2]
```diff
@@ -107,11 +129,17 @@ Feature: sharing
...
     # Section 2: Brian receives and accepts the group share from Alice. Brian now has 2 shares of the same folder owned by Alice
     # The server "merges" the 2 shares and presents them to Brian as a single folder inside the "Shares" folder
     When user "Alice" shares folder "/merge-test-outside-groups-renamebeforesecondshare" with group "grp1" using the sharing API
-    And user "Brian" accepts share "/merge-test-outside-groups-renamebeforesecondshare" offered by user "Alice" using the sharing API
-    Then as "Brian" folder "/Shares/merge-test-outside-groups-renamebeforesecondshare" should exist
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
+    When user "Brian" accepts share "/merge-test-outside-groups-renamebeforesecondshare" offered by user "Alice" using the sharing API
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiShareManagementToShares/mergeShare.feature:143]: same as above

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

**Review by phil-davis — APPROVED** [review]

**Pull request merged**